frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx

```diff
@@ -18,7 +18,9 @@ export interface HogQLQueryEditorProps {
 let uniqueNode = 0
 export function HogQLQueryEditor(props: HogQLQueryEditorProps): JSX.Element {
     const [key] = useState(() => uniqueNode++)
-    const hogQLQueryEditorLogicProps = { query: props.query, setQuery: props.setQuery, key }
+    const [monaco, setMonaco] = useState(null as Monaco | null)
+    const [editor, setEditor] = useState(null as importedEditor.IStandaloneCodeEditor | null)
```

Comment:
Nit: Since these come as a package together, I propose just one `useState` of type `[Monaco, importedEditor.IStandaloneCodeEditor] | null`